### ISSUE TITLE
Loss becomes NaN when training IBQ

### ISSUE THREAD

**Issue by szhcz** [TencentARC/SEED-Voken#53]
Hi, I encountered an issue when training IBQ. I kept the original code unchanged but only reduced the batch size. After running bash scripts/train_tokenizer/IBQ/run_16384.sh for about 19.5 hours, the loss became NaN.
Thanks in advance!

**Comment by ShiFengyuan1999** [comment]
Hi @szhcz, did you keep the learning rate unchanged when reducing the batch size? If so, you might consider lowering it to a smaller value, such as 5e-5, which tends to be more stable for small batch size scenarios.
